pkg/apis/pipeline/v1/pipeline_types.go

```diff
@@ -352,9 +353,41 @@ func (pt *PipelineTask) GetMatrixCombinationsCount() int {
 	if !pt.IsMatrixed() {
 		return 0
 	}
+	// Explicit combinations in the Matrix
+	// Calculate based on length of include Names and return count
+	if pt.Matrix.MatrixHasInclude() && !pt.Matrix.MatrixHasParams() {
+		return len(pt.Matrix.Include)
+	}
+	// Create a set of param names
```

Comment:
suggest removing this comment: it's a bit confusing because this variable is a map, and doesn't help explain the "why" (it just describes the next line of code)